pkg/db/kv_migration_test.go

```diff
@@ -17,7 +17,9 @@ func TestKVMigration(t *testing.T) {
 	ctx := context.Background()
 	dbParams := params.Database{Driver: "pgx", ConnectionString: databaseURI, KVEnabled: true, DropTables: true}
 	kvStore, err := kv.Open(ctx, kvpg.DriverName, kvparams.KV{Postgres: &kvparams.Postgres{ConnectionString: dbParams.ConnectionString}})
+	require.NoError(t, err)
```

Comment:
@nopcoder uses `require`. Now I've seen everything 😄

Reply:
ha :D used the same code the test implementor used - still prefer the simple code.